src/DetailsView/components/report-export-component.tsx

```diff
@@ -103,11 +103,10 @@ export class ReportExportComponent extends React.Component<
     };
 
     public render(): JSX.Element {
-        const { deps, reportExportFormat, isOpen, featureFlagStoreData } = this.props;
+        const { reportExportFormat, isOpen, featureFlagStoreData } = this.props;
 
         return (
             <ExportDialog
-                deps={deps}
```

Comment:
same as above?

Reply:
Yes, tested and deps property was removed from its interface ExportDialogProps in its previous commit.